grails-app/conf/Config.groovy

```diff
@@ -1804,6 +1804,45 @@ openboxes.configurationWizard.listOfDemoData = [
                  "<ul>",
 ]
 
+openboxes.configurationWizard.dataInit = [
```

Comment:
Can we convert this to Groovy DSL syntax
```
openboxes.configuration.data { 
    locations { 
        enabled = true
        url = 
    }
    ...
}
```

Reply:
Yup, agree. I gave the same suggestion.

Reply:
done

Reply:
I reverted change to DSL as I was not able to use it as key-value map

Reply:
It works, thanks